src/main/java/com/bravo/user/dao/repository/PaymentRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.bravo.user.dao.repository;
+
+import com.bravo.user.dao.model.Payment;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface PaymentRepository extends JpaRepository<Payment,String> {
+
+    // Finds All Payments linked to the User ID provided
```

Comment:
Do you think this comment adds value?

Reply:
Not really a lot. More so to briefly describe what the method will do. 
I could have gone into more detail of how it works based on the method signature and name, being that the way this works is somewhat abstract.